settings.gradle.kts

```diff
@@ -14,4 +18,7 @@ sourceControl {
     gitRepository(URI.create("https://github.com/sfa-siard/JdbcBase.git")) {
         producesModule("ch.admin.bar:JdbcBase")
     }
+    gitRepository(URI.create("https://github.com/sfa-siard/JdbcBase.git")) {
+        producesModule("ch.admin.bar:jdbc-base")
+    }
```

Comment:
remove one of the jdbc base git repositories - keep `jdbc-base` - it just looks nicer